languages/elixir/exercises/concept/tuples/.docs/hints.md

```diff
@@ -0,0 +1,43 @@
+### General
+
+- [Tuples][tuple-module] are data structures which are arranged in contiguous memory and can hold any data-type.
+- Atoms may be used to denote finite states, as this exercise uses `:cup`, `:fluid_ounce`, `:teaspoon`, `:tablespoon`, `:millilitre` to denote the units used.
+- You may use `Kernel` or `Tuple` functions or pattern matching to manipulate to manipulate the tuples.
+
+### 1. Implement the `get_volume/1` function
+
+- Consider using a `Kernel` module function to return the volume-pair's numeric component.
+- Remember, one-line functions are best used for short elixir functions.
+
+### 2. Implement the `to_millilitre/1` functions
+
+- Use [multiple clause functions][multi-clause] and pattern matching to reduce conditional control flow logic.
+
+  ```elixir
+  defmodule Example do
+    def foo(:bar), do: true
+    def foo(:baz), do: true
+    def foo(_), do: false
+  end
+
+  Example.foo(:bar)
+  # => true
+  Example.foo(:baz)
+  # => true
+  Example.foo(:blah)
+  # => false
+  ```
```

Comment:
We usually try to not have any code in the hints file, but instead only refer to external resources or use descriptions. Does the resource that is linked to above this code not provide enough information?

Reply:
Sure, I'll link to a pattern matching on function clause resource.